client/components/select-vertical/suggestion-search.tsx

```diff
@@ -134,12 +140,38 @@ const SelectVerticalSuggestionSearch: FC< Props > = ( {
 		] );
 	}, [ translate, suggestions, isLoading, isShowSuggestions, isShowSkipOption ] );
 
+	useEffect( () => {
+		if ( ! ( window.visualViewport && isMobile ) ) {
+			return noop;
```

Comment:
hmm do we need to return `noop`? I thought we could just `return;`

Reply:
I remembered that `eslint` will complain about [the inconsistent return](https://eslint.org/docs/rules/consistent-return). But I found that we have disabled that rule, I'll change to `return;` only 👍